src/client/public/styles/nav.css

```diff
@@ -1,11 +1,16 @@
 nav-element {
-  width: 250px;
+  width: var(--nav-size);
```

Comment:
--nav-size is defined twice, once in the root and once in the element, would it make sense to use params with a hardcoded fallback? `width: var(--nav-size, 250px);`

Reply:
Fixed.